device/dhcp/servers/v4/dhcpv4serverleases.yaml

```diff
@@ -0,0 +1,91 @@
+components:
+  schemas:
+    DhcpServerV4.Pool:
+      description: >-
+        Configuration for DHCPv4 address pool for a lease.
+      type: object
+      x-status:
+        status: under_review
+      required: [name, start_address]
+      properties:
+        name:
+          x-include: ../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        lease_time: 
+          description: >-
+            The Life Time length in seconds that is assigned to a lease.
+          type: integer
+          format: uint32
+          minimum: 10
+          default: 86400
+          x-field-uid: 2
+        start_address:
+          description: >-
+            The IP address of the first lease pool.
+          type: string
+          format: ipv4
+          x-field-uid: 3
+        prefix:
+          description: >-
+            The IPv4 network prefix length to be applied to the address. 
+          type: integer
+          format: uint32
+          default: 24
+          maximum: 32
+          x-field-uid: 4
+        count:
+          description: >-
+            The total number of addresses in the pool.
+          type: integer
+          format: uint32
+          default: 1
+          minimum: 1
+          x-field-uid: 5
+        step:
+          description: >-
+            The increment value for the lease address within the lease pool.
+            addresses are present. The value is incremented according to the Prefix
+            Length and Step.
```

Comment:
"addresses are present" looks like a typo. 
according to the prefix[_length] and step. [ Use same names as the attributes ]